### ISSUE TITLE
Doc review

### ISSUE THREAD

**Issue by pibi** [expertontheclock/www#6]
EOTC-Application specs [here](https://github.com/expertontheclock/www/wiki/EOTC-Application)

**Comment by cw6365** [comment]
Looks great and correct Ivano. One thing to bare in mind when we build, the right sidebar which has coming soon etc.. how i'm going to do this in version 2 is to have an integration plugin system like slack where companies (and us) can develop integrations for the system which experts can activate, eg Google Drive or A terminal.

**Comment by pibi** [comment]
Yep. This is why we used the "sidepanel component" as an abstraction. We need to further investigate how to model it to be "plug and play", but with component and with AngularJS native support to dependency injection we should have all the basic pieces in place to support the "plugins"

**Comment by cw6365** [comment]
Just to let you know i've got the early POC set up locally now so if you need me for anything let me know.

**Comment by pibi** [comment]
Hi Chris,

I’m Max, the guy who is working with Ivano on your project. Here is a summary of the actual work: we done the basic port and started the componentization of the frontend. Now we are fighting the CSS and structure of your HTMLs, trying to fit them on the Bootstrap grid system. The Bootstrap grid lets us to be cross browser easily and also ease the development of a complete responsive website. The original HTMLs lack a correct grid implementation.
For example in the account* files the bootstrap class col-* is used but not inside the row and container classes; in the files under connection-view folder the bootstrap grid system classes are not used at all. In this way, as you can see from the screenshots, the layout is quite not responsive
(the left side doesn't resize well with the browser window, and the layout is broken).

![eotc-fullpage](https://cloud.githubusercontent.com/assets/191260/10710968/451a374a-7a6d-11e5-91d8-de1a502a0b11.png)

![eotc-resized](https://cloud.githubusercontent.com/assets/191260/10710970/62586714-7a6d-11e5-8dcf-7acaf0105656.png)

In our POC instead, the left side moves accordingly with the browser and the results is more responsive (the layout didn’t broke).

![poc-fullpage](https://cloud.githubusercontent.com/assets/191260/10710975/9e614762-7a6d-11e5-9862-9b6a0d5d8863.png)

![poc-resized](https://cloud.githubusercontent.com/assets/191260/10710976/a66476dc-7a6d-11e5-85ac-6cd2dadb5943.png)

We need to fix this now, because it’s easier and cheaper than do the layout fixes later.

You can check the actual code on the EOTC_CSSPORT branch. As you can see on the screenshot, we are experiencing minor issue on firefox when full-sized, maybe by some fixed padding values on the CSS. 

(firefox)
![eotc-last-firefox-fullpage](https://cloud.githubusercontent.com/assets/191260/10711042/485bd596-7a70-11e5-90ef-16c26b78d769.png)

![eotc-last-firefox-resized](https://cloud.githubusercontent.com/assets/191260/10711045/50b0fdde-7a70-11e5-9f6e-1388a3035a6b.png)

(chrome)
![eotc-last-chrome-fullpage](https://cloud.githubusercontent.com/assets/191260/10711051/709b543c-7a70-11e5-9b1d-3fa18a53fa9b.png)

![eotc-last-chrome-resized](https://cloud.githubusercontent.com/assets/191260/10711053/78e38114-7a70-11e5-9542-fc8e9b74183f.png)

We are still working on it, but if you want to take a look at the code and gives to us feedback or hints, we will really appreciate it.

Cheers,

Max

**Comment by cw6365** [comment]
Hi Max

Yes that's fine. I'll take a look, we can test and adjust the html as we go too. The layout isn't going to be the best anyway for small viewports, we're going to have a dedicated app for smaller devices but yes if you can fix where possible or assign it to us.

**Comment by pibi** [comment]
hi chris,
we have just closed the "CSS porting issue". Now it is as responsive as the POC. we tested it on chrome, firefox and IE.
cheers.

**Comment by cw6365** [comment]
Great work, thanks
